### PR TITLE
Test One

### PR DESCRIPTION
**Pull request by Gegardus**
This is a unit tests project for the To-Do list application. The testing project is divided into two parts. In this part are tested functions responsible for adding and deleting list items.

### CODE REVIEW

File: addRemove.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import TodoList from './src/module/updateStatus.js';
```
Comment (GzimAsani) [inline addRemove.test.js:1]: - [x] Good job testing the add and delete files but you should also used mock for funtions that used localStorage, DOM or external APIs

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @Gegardus and @Banstein                                      
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this 
You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: addRemove.test.js (hunk 1 above)
Comment (Gegardus) [reply addRemove.test.js:1]: Hi @GzimAsani DOM and external API should be tested in the second part of the project. I have mentioned that in a comment.

Comment (GzimAsani) [reply addRemove.test.js:1]: Hi, @Gegardus, sorry for my mistake

Comment (Gegardus) [reply addRemove.test.js:1]: Hi @GzimAsani what we can do for now

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gegardus** [comment]
Thank you.

**Pull request closed without merging**